downstream/modules/platform/ref-controller-search-tips.adoc

```diff
@@ -3,15 +3,15 @@
 = Rules for searching
 
 These searching tips assume that you are not searching hosts. 
-Most of this section still applies to hosts but with some subtle differences. 
+//Most of this section still applies to hosts but with some subtle differences. 
 
-* The typical syntax of a search consists of a field (left-hand side) and a value (right-hand side). 
+* The typical syntax of a search consists of a field (left side) and a value (right side). 
```

Comment:
Instead of left-hand side and right-hand side, maybe 'a field, followed by a value' ?